File: bench/Reference/file_write_read.js
const fs = require('fs');
const data = fs.readFileSync('bench/data/bench_text.txt', 'utf8');
fs.writeFileSync('/tmp/yona_bench_write_js.txt', data);
const data2 = fs.readFileSync('/tmp/yona_bench_write_js.txt', 'utf8');
console.log(data2.length);

File: bench/Reference/file_write_read.java
import java.nio.file.*;
class file_write_read {
    public static void main(String[] args) throws Exception {
        String data = Files.readString(Path.of("bench/data/bench_text.txt"));
        Files.writeString(Path.of("/tmp/yona_bench_write_java.txt"), data);
        String data2 = Files.readString(Path.of("/tmp/yona_bench_write_java.txt"));
        System.out.println(data2.length());
    }
}
